packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -278,8 +299,10 @@ export default function PostOptionsMenu({
     },
     {
       icon: <MenuIcon Icon={BlockIcon} />,
```

Comment:
I would say we should have 2 different icons, one for blocking and an opposite for unblocking. Otherwise it's quite confusing to the user, that we say `Show posts from`, but show the same icon as for `Not interested in...` actions.

Reply:
I thought about leaving the same feedback, but see we already always use the same even for existing block/unblocking actions.
Can probably be something to raise separate to design team.

Reply:
Also, this is a pre-existing issue where the state is not changed even after the action.